src/portal/src/app/project/p2p-provider/task-list/task-list.component.html

```diff
@@ -76,25 +76,34 @@ <h2 class="custom-h2 h2-style">{{executionId}}</h2>
 
   <div class="tasks-detail">
     <h3 class="modal-title">{{'P2P_PROVIDER.TASKS' | translate}}</h3>
-    <div class="row flex-items-xs-between flex-items-xs-bottom">
-      <div class="action-select">
-        <span class="refresh-btn">
-          <clr-icon shape="refresh" (click)="refreshTasks()"></clr-icon>
-        </span>
-      </div>
-    </div>
     <clr-datagrid (clrDgRefresh)="clrLoadTasks(true)" [clrDgLoading]="loading">
-      <clr-dg-column [clrDgField]="'id'">{{'REPLICATION.TASK_ID'| translate}}</clr-dg-column>
-      <clr-dg-column [clrDgField]="'execution_id'">{{'P2P_PROVIDER.ID' | translate}}</clr-dg-column>
-      <clr-dg-column [clrDgField]="'status'">{{'REPLICATION.STATUS' | translate}}</clr-dg-column>
+      <clr-dg-action-bar>
+        <div class="row flex-end">
+          <div class="select filter-tag clr-select-wrapper" [hidden]="!isOpenFilterTag">
+            <select id="selectKey" (change)="selectFilterKey($event)">
+              <option value="id">{{"REPLICATION.TASK_ID" | translate | lowercase}}</option>
+              <option value="execution_id">{{"P2P_PROVIDER.ID" | translate | lowercase}}</option>
+              <option value="status">{{"REPLICATION.STATUS" | translate | lowercase}}</option>
+            </select>
+          </div>
+          <hbr-filter (filterEvt)="doFilter($event)" [currentValue]="searchString" id="filter-executions" [withDivider]="true"
+                      (openFlag)="openFilter($event)" filterPlaceholder='{{"REPLICATION.FILTER_PLACEHOLDER" | translate}}'></hbr-filter>
+          <span class="refresh-btn">
+                                 <clr-icon shape="refresh" (click)="refreshTasks()" [hidden]="loading"></clr-icon>
```

Comment:
The refresh icon  refreshes policy table not task table.

Reply:
Fixed.